### PR TITLE
remove unused emits from all_forms/views

### PR DESCRIPTION
**Pull request by dannyroberts**
reduces view size by 63% (were 16 emits, now there are 6)

This is a step on towards https://trello.com/c/XfVvd7v4/31-all-forms-view-couch-view. Rewriting all the places that are using this view is a little bigger (though manageable) and we can have this benefit right now with a reindex, so I thought I'd put it out there and see what you think.

@czue

### CODE REVIEW

**Comment by dannyroberts** [comment]
@gcapalbo buddy

**Comment by gcapalbo** [comment]
looks ok to me. just to doublecheck - you've made sure nothing uses the 'completion' emits or the 'submission app user' or 'submission xmlns app user' emits?

**Comment by dannyroberts** [comment]
Yeah that's right. I searched the code for `"completion` and `'completion`, and also made sure nothing was calling `make_form_couch_key` with `by_submission_time=False`; for the other two, I took note of what was being used in every call to `all_forms/view` and they weren't ever used.

**Comment by czue** [comment]
very nice! seems worth it to me to do this now though i have no idea how long it will take to reindex.

**Comment by gcapalbo** [comment]
@dannyroberts sounds like we're all signed off on this so feel free to self-merge and kick off the preindex when you're ready

File: corehq/couchapps/all_forms/views/view/map.js [hunk 1]
```diff
@@ -40,23 +40,14 @@ function(doc) {
...
-                    emit([status+" xmlns",          doc.domain, xmlns,      times[status]], emit_entry);
-                    emit([status+" app",            doc.domain, app_id,     times[status]], emit_entry);
-                    emit([status+" user",           doc.domain, user_id,    times[status]], emit_entry);
-                    emit([status+" app user",       doc.domain, app_id, user_id, times[status]], emit_entry);
-                    emit([status+" xmlns app",      doc.domain, xmlns, app_id,  times[status]], emit_entry);
-                    emit([status+" xmlns user",     doc.domain, xmlns, user_id, times[status]], emit_entry);
-                    emit([status+" xmlns app user", doc.domain, xmlns, app_id, user_id, times[status]], emit_entry);
-                }
-            }
+            var status = 'submission';
```
Comment (emord) [inline corehq/couchapps/all_forms/views/view/map.js]: nit: This doesn't look used anymore

Comment (dannyroberts) [inline corehq/couchapps/all_forms/views/view/map.js]: right, thanks

**Comment by esoergel** [comment]
This is awesome!
:star:

**Pull request merged**